packages/ui/src/components/Snackbar.tsx

```diff
@@ -0,0 +1,38 @@
+import { Stack, Snackbar as MuiSnackbar, styled } from '@mui/material'
+import { useToasts } from '../contexts/ToastContext'
+import ToastBar from './ToastBar'
```

Comment:
We can group all the files ToastBar, ToastContent and Snackbar in one folder as it's a one component

Reply:
done